requirements/base.in

```diff
@@ -1,5 +1,8 @@
 # Anything above "--- END OF CUSTOM SECTION ---"
 # will not be touched by ``make_base.py``
+
+h5py
+
```

Comment:
Added it in base because it is needed to run the script

Reply:
Should it be an optional dependency?

Reply:
Dependencies of the package are defined in `pyproject.toml`. `base.in` is just for CI.

I think this should NOT be a dependency of the package (even though making it optional is an option), so not touching `pyproject.toml` is fine. But unless we want to script to be runnable in CI jobs other than the tests, I think it should be added in `basetest.in`, not `base.in`.

That being said: Since the package depends on ScippNexus, which pulls in h5py I do not think you actually have to add it anywhere?